graph/floydwarshall_test.go

```diff
@@ -0,0 +1,109 @@
+package graph
+
+import (
+	"math"
+	"testing"
+)
+
+//noConnection is a value (INF) which represents that there are no two edges that directly connects these two nodes
+var noConnection = math.Inf(1)
```

Comment:
This is commented out in the actual algorithm file - I think you can remove this and instead write it so that the one in the other file is exported in that package. Also, I would just call it `Inf` and not `noConnection` or `maxValue` as its written in the other file.

Reply:
Thanks for that, removed both `maxValue` and `noConnection` and added `Inf`.